packages/mural-picker/src/components/mural-picker/styles.scss

```diff
@@ -25,10 +25,13 @@
     .workspace-select,
     .room-select {
       flex-grow: 5;
+      min-width: 200px;
```

Comment:
If we move the `min-width` inside the following css at line 20, the min-width will be the same for all children of `mural-picker-selects`. (A `min-width` of 175px if giving good results)
```
& > * {
      margin-top: 0.375rem;
      margin-right: 0.5rem;
      min-width: 175px;
    }
```